### ISSUE TITLE
Update Failed

### ISSUE THREAD

**Issue by Xx-Ness-xX** [SickRage/SickRage#3682]
### Before submitting your issue:

Enable debug logging in SickRage settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit: **master/558cd35a9c7dee197cced35a0db9361de1c41ca3**
OS: **Windows-10-10.0.14393**
What you did: Click **"Update Now"**
What happened: **"Updated Failed" shown**
What you expected: **I should be able to update successfully**
<details>
<summary>Logs:</summary>

```
AA AniDBCommandTimeoutError: Command has timed out
AA raise AniDBCommandTimeoutError("Command has timed out")
AA File "C:\SickRage\lib\adba\__init__.py", line 127, in handle
AA return self.handle(AuthCommand(username, password, 3, self.clientname, self.clientver, nat, 1, 'utf8', mtu), callback)
AA File "C:\SickRage\lib\adba\__init__.py", line 207, in auth
AA resp = self.auth(self._username, self._password)
AA File "C:\SickRage\lib\adba\__init__.py", line 147, in _reAuthenticate
AA if self._reAuthenticate():
AA File "C:\SickRage\lib\adba\__init__.py", line 83, in handle_response
AA self.handle_response(command.resp)
AA File "C:\SickRage\lib\adba\__init__.py", line 129, in handle
AA result = self.handle(LogoutCommand(), callback)
AA File "C:\SickRage\lib\adba\__init__.py", line 214, in logout
AA ADBA_CONNECTION.logout()
AA File "C:\SickRage\sickbeard\__init__.py", line 1726, in halt
AA sickbeard.halt() # stop all tasks
AA File "C:\SickRage\SickBeard.py", line 453, in shutdown
AA self.callback(type)
AA File "C:\SickRage\sickbeard\event_queue.py", line 47, in run
AA Traceback (most recent call last):
2017-04-27 14:57:51 ERROR EVENT-QUEUE :: [558cd35] Exception generated in thread EVENT-QUEUE: Command has timed out
```
</details>

**Comment by sharkykh** [comment]
This indicates an issue with AniDBDB
* Try restarting SickRage (preferably manually)
* Try temporarily disabling AniDB in your config (http://localhost:8081/config/anime/) and try again.

**Comment by Xx-Ness-xX** [comment]
- I tried restarting the SickRage it did not work, still update failed

- I temporarily disabled AniDB in my config, and still update failed

- Then, I tried to manually shut down SickRage, and it showed _SickRage is shutting down..._ but I can still access to SickRage

- Finally, I restarted my computer, and turn on SickRage. Now SickRage shows there is no update so I guess the issue is fixed? I will let you know what I find later

**Comment by sharkykh** [comment]
I think SickRage/AniDB got stuck in some weird limbo not being able to log out or something.
I hope that does fix the issue.

Just so you know, when you install SickRage on Windows it runs using a service,
SickRage can't control services so the Shutdown link on browser **restarts the SickRage**
What I meant by "manually" was [using the links in your start menu](https://cloud.githubusercontent.com/assets/10238474/25532744/0ad8b3f4-2c37-11e7-84c0-5d682840f48e.png)
But restarting your computer is the same thing.

**Comment by Xx-Ness-xX** [comment]
I installed my SickRage using a vbs shortcut according to https://www.htpcguides.com/install-sickrage-windows-usenet-torrent-tv/ ,however, I did not put the vbs shortcut into my Startup folder (personal preference). Thus, manually shut down should have worked.

After I have restarted my computer, I manually started SickRage, then I tried to manually shutdown again. This time it shut down completely, I could not access to my SickRage anymore until I re-run the vbs shortcut.

Like you said, SickRage/AniDB must have gotten stuck in some weird limbo mode that prevent it from shutting down completely.
